### ISSUE TITLE
[BUG] AttributeError: 'series' object has no attribute 'array' when calling np.array(df)

### ISSUE THREAD

**Issue by bingoko** [xprobe-inc/xorbits#248]
### Describe the bug
When calling `np.array(df)` in my code, I encountered the following error: "`AttributeError: 'series' object has no attribute '__array__'`".

### To Reproduce
To help us to reproduce this bug, please provide information below:

1. Your Python version: python 3.8
2. The version of Xorbits you use: the latest, 0.2.0
3. Versions of crucial packages:
numpy==1.22.4
5. Minimized code to reproduce the error.

```
import numpy as np
import xorbits.pandas as pd

# code that triggers the error
data = np.array(df)
```

### Expected behavior
I expected the `np.array()` function to create a numpy array from the Pandas DataFrame `df`.

### Additional context
N/A

**Comment by qianduoduo0904** [comment]
Thank you for your report, we will investigate the error and fix it ASAP.

**Comment by Bojun-Feng** [comment]
Hi, I am new to the codebase and this seems to be a good first issue to work on. If anyone else is working on this issue please let me know, else I'll start working on it tonight.

**Comment by aresnow1** [comment]
> Hi, I am new to the codebase and this seems to be a good first issue to work on. If anyone else is working on this issue please let me know, else I'll start working on it tonight.

Nobody is currently working on it, welcome to contribute :)

**Comment by qianduoduo0904** [comment]
Small tip, you can comment "take" to take this issue:)

**Comment by Bojun-Feng** [comment]
Thank you for the warm feedback and helpful suggestions! I am still relatively new to coding, so please let me know if I'm doing anything wrong and I will fix it ASAP.

**Comment by qianduoduo0904** [comment]
Feel free to reach out to us, we will help you as best as we can.

**Comment by vijaybirju** [comment]
Can tell me which file has this issue?

**Comment by aresnow1** [comment]
> Can tell me which file has this issue?

@Bojun-Feng has submitted a pull request in #282 .

**Comment by qianduoduo0904** [comment]
This issue is addressed on main branch, and will be included in coming v0.3.0, stay tuned!
